editors/vscode/package.json

```diff
@@ -118,7 +118,7 @@
           ],
           "scope": "resource",
           "default": null,
-          "description": "Path to TypeScript configuration. If your `tsconfig.json` is not at the root, alias paths will not be resolve correctly for the `import` plugin."
+          "markdownDescription": "Path to TypeScript configuration. If your `tsconfig.json` is not at the root, alias paths will not be resolve correctly for the `import` plugin."
```

Comment:
Do we auto-find the user's tsconfig file nowadays? Maybe we should update this to note that it's not necessary as long as you have a tsconfig file at the root.

Reply:
No worries! I've updated this description so it's a bit clearer now :)